### PR TITLE
User Settings rename to Account Settings

### PR DESCRIPTION
**Pull request by jillirami**
### What

Changed "user settings" references to "account settings" in the following pages:

- website/docs/cloud-docs/api-docs/github-app-installations.mdx
- website/docs/cloud-docs/users-teams-organizations/2fa.mdx
- website/docs/cloud-docs/users-teams-organizations/organizations.mdx
- website/docs/cloud-docs/users-teams-organizations/single-sign-on/linking-user-account.mdx
- website/docs/cloud-docs/users-teams-organizations/users.mdx

- website/docs/cloud-docs/api-docs/policy-sets.mdx
- website/docs/cloud-docs/api-docs/private-registry/modules.mdx
- website/docs/cloud-docs/api-docs/workspaces.mdx
- website/docs/cloud-docs/users-teams-organizations/api-tokens.mdx
- website/docs/cloud-docs/users-teams-organizations/single-sign-on/index.mdx

### Why

Updated any docs that refer to User settings, to rename User settings to Account Settings. We are renaming this documentation to state Account Settings, as this is being changed in our code for all users via https://hashicorp.atlassian.net/browse/TF-11668

### Screenshots

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [x] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [x] Description links to related pull requests or issues, if any.

#### Content
- N/A - Redirects have been added to `website/redirects.js` for moved, renamed, or deleted pages.
- [ ] API documentation and the API Changelog have been updated. 
- N/A - Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- N/A - Pages with related content are updated and link to this content when appropriate.
- N/A - Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- N/A - New pages have metadata (page name and description) at the top.
- N/A - New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- N/A - New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [x] UI elements (button names, page names, etc.) are bolded.
- [x] The Vercel website preview successfully deployed.

#### Reviews
- [x] I or someone else reviewed the content for technical accuracy.
- [x] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

**Review by haydengomes — APPROVED** [review]
Generally looks good to me. There's a checklist in the description that I'm unfamiliar with but would be good to go through before merging

**Review by haydengomes — APPROVED** [review]

File: website/docs/cloud-docs/users-teams-organizations/2fa.mdx [hunk 1]
```diff
@@ -13,7 +13,7 @@ User accounts can be additionally protected with two-factor authentication (2FA)
 
 ## Setting up Two-factor Authentication
 
-To reach your user security settings page, click the user icon in the upper right corner and choose "User Settings" from the menu.
+To reach your user security settings page, click the user icon in the upper right corner and choose "Account Settings" from the menu.
```
Comment (rkoron007) [inline website/docs/cloud-docs/users-teams-organizations/2fa.mdx:16]: This is a pro-active suggestion outside of your change, but we use bold with UI elements in the docs, so appreciate if we tweak this to:
```suggestion
To reach your user security settings page, click the user icon in the upper right corner and choose **Account Settings** from the menu.
```

**Review by rkoron007 — APPROVED** [review]
Looks great to me; thanks for handling this so thoroughly! ✨

**Comment by jillirami** [comment]
I will merge this change along with https://github.com/hashicorp/atlas/pull/18336 (once it is ready for merge)

**Review by haydengomes — APPROVED** [review]

**Pull request merged**